tests/dummy/app/templates/components/color-label.hbs

```diff
@@ -1,4 +0,0 @@
-<label style="color: green" for={{inputId}} class={{config.css.label}}>
-  {{yield}}
-  Custom label component!
-</label>
```

Comment:
Should this really be deleted?

Reply:
IMHO, yes. A custom label component is not something you'd want to show off in a demo app. It's enough to explain it in the readme.

Reply:
Also it looked weird.

Reply:
:smile: ok